### ISSUE TITLE
Data error (cyclic redundancy check).

### ISSUE THREAD

**Issue by TayDzn** [DurtyFree/durty-cloth-tool#345]
I have the following error:
```
System.IO.IOException: Data error (cyclic redundancy check).
   at Microsoft.Win32.SafeHandles.SafeFileHandle.OverlappedValueTaskSource.GetResult(Int16 token)
   at System.IO.Strategies.BufferedFileStreamStrategy.ReadAsyncSlowPath(Task semaphoreLockTask, Memory`1 buffer, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.PoolingAsyncValueTaskMethodBuilder`1.StateMachineBox`1.System.Threading.Tasks.Sources.IValueTaskSource<TResult>.GetResult(Int16 token)
   at System.Security.Cryptography.HashAlgorithm.ComputeHashAsyncCore(Stream inputStream, CancellationToken cancellationToken)
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.Nf3BtPV1pi(String  )
   at zqZfS7hdsjq35d65mOGI.c9ysrR0Ya75(Object , zqZfS7hdsjq35d65mOGI )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.Nf3BtPV1pi(String  )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.<>c__DisplayClass38_0.x1Umiws0j4VjQB1vsppc.MoveNext()
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_1(Object state)
   at System.Threading.QueueUserWorkItemCallback.<>c.<.cctor>b__6_0(QueueUserWorkItemCallback quwi)
   at System.Threading.ExecutionContext.RunForThreadPoolUnsafe[TState](ExecutionContext executionContext, Action`1 callback, TState& state)
   at System.Threading.QueueUserWorkItemCallback.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()
   at System.Threading.Thread.StartCallback()
```

App Version: **3.0.0.8**
Project targets: **FiveM**
DLC name: **mpcivilian2**
Collection name: **mpcivilian2 // **
Project path: **D:\Users\Owen\Desktop\GTA\Durty Cloth Tool\clothproject_mpcivilian2**
Output path: **D:\Users\Owen\Desktop\GTA\Durty Cloth Tool**
Disable Extra Ymt Creation: **False**
Create Subfolder For Cloth Drawable Types: **True**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **5**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Hi @TayDzn can you reproduce this issue and if yes can you pls send me your cloth project export via DM ?

**Comment by DurtyFree** [comment]
Fixed next update

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0

**Comment by TayDzn** [comment]
Still happened after update so i just rebuilt the pack
